src/framework/control.ml

```diff
@@ -29,10 +29,13 @@ let get_spec () : (module Spec) =
 module AnalyzeCFG (Cfg:CfgBidir) =
 struct
 
-  (** The main function to preform the selected analyses. *)
-  let analyze (file: file) (startfuns, exitfuns, otherfuns: Analyses.fundecs)  (module Spec : Spec) =
+    (** The main function to preform the selected analyses. *)
+    let analyze (file: file) (startfuns, exitfuns, otherfuns: Analyses.fundecs)  (module Spec : Spec) (increment: increment_data) =
+  
+    let module Inc = struct let increment = increment end in
+
     (** The Equation system *)
-    let module EQSys = FromSpec (Spec) (Cfg) in
+    let module EQSys = FromSpec (Spec) (Cfg) (Inc) in
```

Comment:
Why do we need this module?

Reply:
So far only needed to access `S.increment` in `td3.ml`.
We should move most of the incremental code out from `td3.ml` into `control.ml`.
`solve` should just get the loaded state and some change information to know what to destabilize.